src/client/components/LeftBlueBar/LeftBlueBar.component.js

```diff
@@ -11,16 +12,17 @@ function SidebarMenu() {
       <div className="sidebar-menu">
         <div className="sidebar-header">
           <div className="sidebar-title">GuideIT</div>
-          <div className="create-account">
+          {/* <div className="create-account">
```

Comment:
If we are not using this anymore, then it is better that you erase it.

Reply:
Thank You @valmdz , I made all the changes as you requested

Reply:
Seems good @rajanibusani